app/src/main/java/org/greenstand/android/TreeTracker/walletselect/WalletSelectScreen.kt

```diff
@@ -77,6 +80,26 @@ fun WalletSelectScreen(
                         }
                     }
                 },
+                centerAction = {
+                        DepthButton(
+                            onClick = { navController.navigate(NavRoute.SignupFlow.route) },
+                            shape = DepthSurfaceShape.Circle,
+                            colors = AppButtonColors.UploadOrange,
+                            modifier = Modifier
+                                .align(Alignment.Center)
+                                .size(height = 70.dp, width = 70.dp),
+                        ) {
+                            Text(
+                                buildAnnotatedString {
+                                    withStyle(style = SpanStyle(color = Color.Black,fontWeight = FontWeight.Bold, fontSize = 50.sp)) {
+                                        append("+")
+                                    }
+
+                                     }
+                            )
```

Comment:
Why build an annotated string? A regular string can work, the text object takes a style.